docs/src/learning_rate_scheduler.md

```diff
@@ -0,0 +1,69 @@
+- ``T\to \infty``: Though theoretical concerns have been made with respect to continuation beyond ``T=1`` for inversion methods such as EKI and UKI, in practice we commonly see better estimates ``\bar{\theta}_{N_\mathrm{it}}`` to the true parameters. As expected this procedure leads to ensemble collapse, and so no meaningful information can be taken from the posterior spread.
+
+## The experiment
+
+We assess the schedulers by solving the inverse problem 100 times with different random initial ensembles (using the same data realization for all). Shown below is an example plot of one of these solves with each timestepper.
+
+![Solution ensembles](assets/ensemble_compare_scheduler.png)
+
+*Left: The true model over ``[0,2\pi]`` (black), and the ensembles from different schemes (colors).
+Right: The noisy observation (black), the distribution it was sampled from (gray-ribbon), and the corresponding ensemble-mean approximation given from each scheduler (colors).*
+
+To assess the timestepping we show the convergence plot against the algorithm iteration we measure two quantities.
+- error (solid) is defined by ``\frac{1}{N_{ens}}\sum^{N_{ens}}_{i=1} \| \theta_i - \theta^* \|^2`` where ``\theta_i`` are ensemble members and ``\theta^*`` is the true value used to create the observed data.
+- spread (dashed) is defined by ``\frac{1}{N_{ens}}\sum^{N_{ens}}_{i=1} \| \theta_i - \bar{\theta} \|^2`` where ``\theta_i`` are ensemble members and ``\bar{\theta}`` is the mean over these members.
+
+![Error vs spread](assets/error_vs_spread_over_iteration_compare_scheduler.png)
+
+*Left: the error and spread of the different timesteppers at over iterations of the algorithm for a single run.
+Right: the error and spread of the different timesteppers at their final iterations, averaged from 100 different draws of initial conditions.*
+
+From this plot there are several take-home messages.
+ - Termination at ``T=1``. DMC with termination (purple), closely mimics a small-fixed timestep (green) that finishes stepping at ``T=1``. Both retain more spread than other approaches, and DMC is far more efficient, typically terminating after around 10-20 steps, where fixed-stepping takes 50. We see that (for this experiment) this is a conservative estimate, as continuing to solve (e.g. brown) until later times often leads to a better error while still retaining similar "error vs spread" curves (before inevitable collapse).
+ - Large fixed step (orange). This is very efficient, but can get stuck in local minima when too large, it collapses the ensemble. Note that even on average it gives lower error to the true parameters than DMC. 
```

Comment:
I think that getting stuck in local minima is usually a problem for small timesteps. For large timesteps the problem I think is usually "overshooting" the minimum.

Reply:
Hmm, I wonder if the intuition is backwards for EKI, or maybe our terminology is not clear enough? When the timestep is too large, I might obtain an overly-collapsed ensemble at some value, and it does not move far from this in future iterations. For small steps, I don't think I have observed any "getting stuck" in local minima, may this is due to the noise added in every iteration.

Reply:
Ah yes, that's a good point, I guess it's hard to apply the intuition from gradient descent. This would be interesting to think more about in the future, but we can leave it as you phrased it then.